src/main/java/com/redhat/documentation/asciidoc/extraction/model/LocalDirectoryTarget.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.documentation.asciidoc.extraction.model;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Objects;
+
+public class LocalDirectoryTarget implements Target {
+    private final File directory;
+
+    public LocalDirectoryTarget(File directory) {
+        this.directory = directory;
+    }
+
+    @Override
+    public Path getDirectoryPath() {
+        return directory.toPath();
+    }
+
+    public File getDirectory() {
```

Comment:
Not really necessary, you can get a `File` from a `Path` and `Path` is a little easier to work with.